lib/shared/log/tasks.js

```diff
@@ -3,15 +3,11 @@
 var archy = require('archy');
 var chalk = require('chalk');
 var log = require('gulplog');
-var sortBy = require('lodash.sortby');
 
-function isString(val) {
-  return typeof val === 'string';
-}
-
-function isObject(val) {
-  return typeof val === 'object' && !Array.isArray(val);
-}
+var _ = require('lodash');
```

Comment:
I think people were trying to transition all the gulp dependencies to use `lodash.*` packages, so this should require `lodash.sortby`, `lodash.isstring` and `lodash.isplainobject`